.github/workflows/upload-test.yml

```diff
@@ -12,8 +12,8 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Upload Episode from YouTube To Anchor.Fm
-        uses: Schrodinger-Hat/youtube-to-anchorfm@V0.1.8
+        uses: Schrodinger-Hat/youtube-to-anchorfm@main
```

Comment:
I've got the same doubt and we discuss on discord.

The main reason why main is the right thing to do here is considering that anchor is frequently updated, the versions previous to the main will not work anymore and the user should change every time a new version is release. In this way the user won't have problems even if we have made some fixes because the modification will be taken directly from the main.

Reply:
Yes, I can understand your point @Wabri, but if we are going to merge a bug or a breaking change we will create different issues for the users.

Having a released version is something preferred in those situations, as the users are sure to have the action stable.

Perhaps, the problem is related to changes within APIs or FE that there are not under our control?

Reply:
I propose we leave it with the version.

But we add a section to the readme detailing how to test the latest changes.

Reply:
I agree with @davideimola

Reply:
@abe-101 can you make this edits to the PR so we are able to merge it? thanks

Reply:
I can suggest starting using the version only with the major (e.g. v0 / v1), so we continuously update the major version continuously with the minor versions.